build-in-docker.sh

```diff
@@ -1,16 +0,0 @@
-#!/bin/bash -e
```

Comment:
I still need some script to copy out the two artifacts onto the host filesystem. Shouldn't be difficult with the merged image though...

Reply:
Don't understand. The artifacts equate to the new docker image, don't they?

Reply:
I don't only run this within docker. Sometimes I need to run it directly on the host for some testing, and then I need them extracted :)

Reply:
Moved that to Makefile, as you saw already. Hope that's fine.